feature/bgp/multipath/otg_tests/bgp_multipath_wecmp_lbw_community_test/README.md

```diff
@@ -0,0 +1,69 @@
+# RT-1.52: BGP multipath ECMP support with Link Bandwidth Community
```

Comment:
```suggestion
# RT-1.52: BGP multipath UCMP support with Link Bandwidth Community
```

UCMP = unequal cost multipath

(Note that ECMP == equal cost multipath, but with link bandwidth, the costs are not equal.)